shoes-core/lib/shoes/common/inspect.rb

```diff
@@ -9,7 +9,7 @@ def to_s
       # Example:
       #   (Shoes::App:0x01234abc "Hello")
       def inspect
-        "(#{self.class.name}:#{'0x%08x' % (object_id * 2)}#{inspect_details})"
+        sprintf('(%s:%s%s)', self.class.name, hexy_object_id, inspect_details)
```

Comment:
aha, I see it's about the `%` in the middle. hmm. See [here](https://github.com/bbatsov/ruby-style-guide#sprintf)